### ISSUE TITLE
Config does not work if cssId field is hidden

### ISSUE THREAD

**Issue by aschempp** [oveleon/contao-component-style-manager#40]
If a user does not have access to the cssID field in permissions but only the style manager, the configuration does not fully work. It seems a user can add a class, but when removing it it remains in the cssID field.

**Comment by doishub** [comment]
Hi Andreas,

unless the cssId field is allowed to be edited, the callback methods (load_callback, save_callback) are not listed. 

Is there a case where the user is allowed to operate the StyleManager, but CSS classes are not allowed to be entered independently?

**From my point of view, we would have two options:**

1. we disallow the StyleManager to be managed detached from the cssId. For example with an onsave callback for user groups and a corresponding hint.

2. the storage and preparation of the CSS classes takes place no more over the class fields themselves, but also over the field styleManager. This would mean however a larger expenditure and some methods would have to be rewritten.

That properties are written directly in the cssId field itself, should only occur if a category / CSS group would be removed from the StyleManager set. The reason for this is so that all properties would still be retained for the time being.

**Comment by aschempp** [comment]
It's not that much of a deal, but it was unclear to me. I'm using stylemanager in one project because I don't want my client to "need to remember" CSS classes. Because he's not supposed to enter anything into that field, I removed it from permissions.

I would prefer the style manager to work independently for that reason, but I understand the technical challenge. Regardless if the solution, the current situation is just confusing because one might not notice it does not work. I just enabled the field permission for my client now which isn't a problem.
